### PR TITLE
add option to chooce compress algorithm

### PR DESCRIPTION
**Pull request by FreedomKnight**
add options for gpg argument to set compress algo

### CODE REVIEW

**Comment by alecpl** [comment]
Please, update also doc blocks. See https://github.com/pear/Crypt_GPG/commit/1a74de34cc4638c for an idea where.

**Comment by FreedomKnight** [comment]
Thanks, I already update the doc blocks

File: Crypt/GPGAbstract.php [hunk 1]
```diff
@@ -249,6 +249,7 @@ abstract class Crypt_GPGAbstract
      *                                       will not be executed via destructor.
      * - <kbd>string digest-algo</kbd>     - Sets the message digest algorithm.
      * - <kbd>string cipher-algo</kbd>     - Sets the symmetric cipher.
+     * - <kbd>string compress-algo</kbd>   - Sets the compress algorithm.
```
Comment (alecpl) [inline Crypt/GPGAbstract.php]: "Compression algorithm". Other than that it's ready to merge. Thanks.

Comment (FreedomKnight) [reply Crypt/GPGAbstract.php]: I fixed this typo, thank you

**Pull request merged**